packages/components/resource-pages/ResourcePopOver.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, Popover, PopoverPosition } from '@patternfly/react-core';
+import ResourceLink from '../../utils/generics/resource-link';
+
+const ResourcePopOver: React.FC<ResourcePopOverProps> = ({
+  position,
+  children,
+  label,
+  title,
+}) => {
+  return (
+    <Popover
+      position={PopoverPosition.auto || position}
+      headerContent={title}
+      bodyContent={children}
+      aria-label={title}
+    >
+      <Button variant="link" isInline>
+        {label}
+      </Button>
+    </Popover>
+  );
+};
+
+export const DataResourcesPopOver: React.FC<DataResourcesPopOverProps> = ({
+  label,
+  dataResources,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <ResourcePopOver label={label} title={t('Connected data sources')}>
+      <div className="resource-pop-over">
+        {dataResources?.map((resourceName) => (
+          <ResourceLink
+            link={`/mcgms/resource/noobaa.io~v1alpha1~NamespaceStore/${resourceName}`}
+            resourceName={resourceName}
+            key={resourceName}
+            hideIcon
+          />
+        ))}
+      </div>
+    </ResourcePopOver>
+  );
+};
+
+export const OBCPopOver: React.FC<OBCPopOverProps> = ({
+  label,
+  obcDetails,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <ResourcePopOver label={label} title={t('Connected OBCs')}>
+      <div className="resource-pop-over">
+        {obcDetails?.map((obcObj) => (
+          <ResourceLink
+            link={`/k8s/ns/${obcObj?.ns}/objectbucket.io~v1alpha1~ObjectBucketClaim/${obcObj?.name}`}
```

Comment:
same here replace ```objectbucket.io~v1alpha1~ObjectBucketClaim``` using getGVK and referenceFor utils

Reply:
ack...